mutalyzer/website/templates/description-extractor.html

```diff
@@ -69,8 +120,38 @@
   {% if not errors %}
     <hr>
 
-    <h4>Genomic description</h4>
-    <p><code>g.{{ description }}</code></p>
+    <h4>Input</h4>
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Field</th>
+          <th>Value</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+        {% if method == 'raw_method' %}
+          <td>Reference sequence</td>
+          <td><code>{{ reference_sequence|short(91) }}</code></td>
+        </tr>
+        <tr>
+          <td>Sample sequence</td>
+          <td><code>{{ variant_sequence|short(91) }}</code></td>
+        {% endif %}
+        {% if method == 'refseq_method' %}
+          <td>Reference accession number</td>
+          <td><code>{{ reference_accession_number }}</code></td>
+        </tr>
+        <tr>
+          <td>Sample accession number</td>
+          <td><code>{{ sample_accession_number }}</code></td>
+        {% endif %}
+        </tr>
+      </tbody>
+    </table>
+    
+    <h4>Description</h4>
+    <p><code><a href="/description?description={{ description }}">{{ description|unicode|too_long(115) }}</a></code></p>
```

Comment:
Internal links should use `url_for()`.

More specifically about this construction:

Conceptually it seems a bit weird to provide an HTTP endpoint that does nothing more than print its query parameter (which the client already has). The HTTP roundtrip could be prevented by just opening the description in a new window using javascript.

Also, URLs have a maximum length of about 2K characters (depending on browser).

So I think it would be better to provide the original input instead of the description. Perhaps by having a `plaintext=1` argument for the `description_extractor` view.

This would also make for a nice `curl`/`wget` endpoint to retrieve a description in plaintext.